crates/evm/evm/src/executors/fuzz/mod.rs

```diff
@@ -127,7 +131,9 @@ impl FuzzedExecutor {
                     // to run at least one more case to find a minimal failure
                     // case.
                     let call_res = _counterexample.1.result.clone();
+                    let _logs = _counterexample.1.logs.clone();
```

Comment:
I think this can be one liner as `logs.borrow_mut().extend(_counterexample.1.logs.clone());`

Reply:
@grandizzy thank you for pointing it out, one liner is much cleaner.